src/components/StatusInner.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect, useRef } from "react";
+import { BsCheck2All } from "react-icons/bs/";
+import { BiErrorCircle } from "react-icons/bi";
+import { BsXCircle } from "react-icons/bs/";
+
+const defaultConfig = {
+  closingAnimationDuration: 3000,
```

Comment:
Here, you initialized the animation to be 3000ms. But in `src/index.js` the value of the duration key in the default props is 5000ms. care to explain?

I can see that you've already calculated it in the startTimer function which makes it around 8000ms in total. I think that duration is a bit too long. what do you think?

Reply:
Hey @kaf-lamed-beyt, so the value in `defaultConfig.closingAnimationDuration` that is 3 seconds mirrors the animation duration in `status.scss` i.e the closing animation (scaling down). Whereas the duration props default value is how long the status stays open before we attempt to close it by playing the closing animation, then immediately after the closing animation the status is completely removed from the DOM.

I think the duration looks good, maybe we run it then tweak the value till we find something that works.

Therefore, the sum is basically after staying open for 5 seconds (enough time to read the message), start the closing animation (3s long). So after the animation ends (5+3), at the 8th second delete the status completely.

oh, and this behavior comes from the autoClose props which is true by default if not the status lingers until the user clicks close.